### PR TITLE
N9kv Support

### PR DESCRIPTION
**Pull request by carlmontanari**
* Adds support for N9kv
  * I'm fairly sure the bios file is not necessary now but I very much recall it being "necessary" before... I do know it does not *hurt* so figured id leave it in...
* Also fixes a screw up from yesterday for the "gen_nics" delay fixup (CLAB_ENV_INTFS vs CLAB_INTFS)

### CODE REVIEW

File: n9kv/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+This is the vrnetlab docker image for Cisco Nexus 9000v virtual switch.
+
+
+## Building the docker image
+
+Put the .qcow2 file in this directory and run make docker-image and you should be good to go. The resulting image is 
+called vr-nxos. You can tag it with something else if you want, like my-repo.example.com/vr-nxos and then push it to 
+your repo. The tag is the same as the version of the NXOS image, so if you have nxosv.9.2.4.qcow2 your final docker 
+image will be called vr-nxos:9.2.4
```
Comment (hellt) [inline n9kv/README.md:11]: that should be `vr-n9kv` I guess

File: n9kv/docker/launch.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        )
+
+        # configure mgmt interface
+        self.wait_write("interface mgmt0")
+        self.wait_write("ip address 10.0.0.15/24")
+        self.wait_write("exit")
+
+        # setup nxapi/scp server
+        self.wait_write("feature scp-server")
+        self.wait_write("feature nxapi")
```
Comment (hellt) [inline n9kv/docker/launch.py:145]: does it have `feature netconf`
or `feature gnmi` maybe (or grpc?)

File: n9kv/docker/launch.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+    def bootstrap_config(self):
+        """Do the actual bootstrap config"""
+        self.logger.info("applying bootstrap configuration")
+        self.wait_write("", None)
+        self.wait_write("configure")
+        self.wait_write(
+            f"username {self.username} password 0 {self.password} role network-admin"
+        )
+
```
Comment (hellt) [inline n9kv/docker/launch.py:137]: I don't see if you have `hostname` provisioned anywhere?

File: n9kv/docker/launch.py (hunk 2 above)
Comment (carlmontanari) [reply n9kv/docker/launch.py:146]: added grpc and netconf!

File: n9kv/docker/launch.py (hunk 3 above)
Comment (carlmontanari) [reply n9kv/docker/launch.py:138]: fixed

File: n9kv/README.md (hunk 1 above)
Comment (carlmontanari) [reply n9kv/README.md]: good catch, fixed!

**Comment by hellt** [comment]
looks good, merging?

**Comment by carlmontanari** [comment]
>looks good, merging? 

Ship it! 😁

**Pull request merged**